src/applications/appeals/shared/components/AreaOfDisagreement.jsx

```diff
@@ -110,8 +118,7 @@ const AreaOfDisagreement = ({
         focusOnChange(scrollKey, 'va-button', 'button');
         updatePage(data);
       } else {
-        // replaces scrollToFirstError()
-        focusOnChange(scrollKey, ERROR_ELEMENTS.join(','));
+        scrollToFirstError({ focusOnAlertRole: true });
```

Comment:
All the changes in this file are also in the [HLR PR](https://github.com/department-of-veterans-affairs/vets-website/pull/32703) - included to keep each PR independent